answers.txt

```diff
@@ -0,0 +1,19 @@
+ Mention two parts of Express that you learned about this week.
+
+A. The two parts of express that I learned about this week are routing and middleware support.
+
+ Describe Middleware?
+
+A. Middleware provides a way to extend features provided by Express such as error handling or security.
+
+ Describe a Resource?
+
+A. A resource is an object with an associated callback.
```

Comment:
A resource is like the actual data we are storing in our database, it is an object indeed, but it does not have to have a callback, in this project your resources will be 'projects' and 'actions' for example.